src/main/webapp/json/news.json

```diff
@@ -1,12 +1,12 @@
 {
-  "articles": [
-    {
+  "articles": {
+    "Black Lives Matter" : [{
```

Comment:
Is the idea that there are multiple articles for each keyword?

Reply:
Yeah